Makefile

```
@@ -125,6 +125,9 @@ resetdb: .state/docker-build-base
 initdb: .state/docker-build-base .state/db-populated
 	$(MAKE) reindex
 
+inittuf: .state/docker-build-base
```

```suggestion
inittuf: .state/db-migrated
	docker compose up -d rstuf-api
	docker compose up -d rstuf-worker
```

this will ensure the db is available so rstuf-worker doesn't crash out (aside: this service should be more resilient and retry connection), then start the services so there's no need to make serve before running `make inittuf`